src/components/Tooltip.tsx

```diff
@@ -74,6 +74,7 @@ export const Tooltip = ({ children, label, long = false }: TooltipProps) => {
 
   return (
     <div
+      style={{ display: "flex" }}
```

Comment:
let's add a class here, inline styling is better to avoid unless no way out so host can customize without `!important`, we already have `excalidraw-tooltip` class, can we add display flex in that instead ?

Reply:
`excalidraw-tooltip` is a different container, but yes.